### PR TITLE
activated dark and dark red themes

### LINKED ISSUE #8: Bunkerchan-like theme or a dark theme

### PR DESCRIPTION
**Pull request by Barbara-Pitt**
Fixing issue #8
Activate 'dark' and 'dark red' themes in config. watch out for cache for these options to show up

### CODE REVIEW

**Review by towards-a-new-leftypol — APPROVED** [review]
Looks good

File: inc/config.php [hunk 1]
```diff
@@ -930,7 +930,8 @@
 	$config['stylesheets']['Yotsuba B'] = ''; // Default; there is no additional/custom stylesheet for this.
 	$config['stylesheets']['Yotsuba'] = 'yotsuba.css';
 	// $config['stylesheets']['Futaba'] = 'futaba.css';
-	// $config['stylesheets']['Dark'] = 'dark.css';
+	$config['stylesheets']['Dark'] = 'dark.css';
```
Comment (towards-a-new-leftypol) [inline inc/config.php]: Let's keep to editing instance-config.php rather than this file

**Review by towards-a-new-leftypol — CHANGES_REQUESTED** [review]

**Comment by towards-a-new-leftypol** [comment]
also it's not up to the devs which themes to use to be honest. I would run it by people first

**Comment by Barbara-Pitt** [comment]
> also it's not up to the devs which themes to use to be honest. I would run it by people first

the users are the ones who requested a "Bunkerchan-like theme or a dark theme" as per issue #8 and feedback in the tech team general thread. This does not change the default theme but merely gives them the option to switch to a dark theme.

File: inc/config.php (hunk 1 above)
Comment (Barbara-Pitt) [reply inc/config.php]: changed the theme changes to instance config and reverted changes to config.php

**Review by towards-a-new-leftypol — APPROVED** [review]

**Pull request merged**